Plotter/python/plot/Plot.py

```diff
@@ -689,7 +689,7 @@ def setaxes(self, *args, **kwargs):
     #frame.GetXaxis().SetLimits(xmin,xmax)
     frame.GetXaxis().SetRangeUser(xmin,xmax)
     frame.SetMinimum(ymin)
-    frame.SetMaximum(ymax)
+    frame.SetMaximum(1.4*ymax)
```

Comment:
Please remove this because it will affect other settings!

If you want to change `ymax`, you should the intended key-word arguments via `Plot/Stack.draw`, either `ymax=...` or `ymargin=1.7`.

Reply:
removed 1.4